koku/api/ingress/reports/serializers.py

```diff
@@ -26,8 +26,10 @@ class Meta:
             "completed_timestamp",
             "reports_list",
             "source",
+            "source_id",
             "bill_year",
             "bill_month",
+            "status",
```

Comment:
I feel like some of these fields should be marked as `read-only=true` right?

https://www.django-rest-framework.org/community/3.0-announcement/#the-extra_kwargs-option

Seems like the uuid, timestamp, and status fields would be, right?